### PR TITLE
Handle request_id globally

### PR DESCRIPTION
**Pull request by vvmnnnkv**
## Description
Improving #715 to return request id in all WS responses including errors.

## Affected Dependencies
n/a

## How has this been tested?
- Unit tests
- Manually tested MC flow

## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [x] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [x] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [x] My changes are covered by tests

### CODE REVIEW

**Comment by vkkhare** [comment]
@vvmnnnkv can you also make it optional for backward compatibility and preventing any changes in pysyft?

File: apps/node/src/app/main/events/__init__.py [hunk 1]
```diff
@@ -66,12 +66,18 @@ def route_requests(message, socket):
     if isinstance(message, bytearray):
         return forward_binary_message(message)
 
+    request_id = None
     try:
         message = json.loads(message)
+        request_id = message.get(MSG_FIELD.REQUEST_ID)
```
Comment (vkkhare) [inline apps/node/src/app/main/events/__init__.py:72]: maybe having a `message.get(request_id,None)` and later check if the request actually supplied it? So we don't want to throw error if the user didn't supply the request id

Comment (vvmnnnkv) [reply apps/node/src/app/main/events/__init__.py:72]: Yes, it's optional. `.get()` returns None by default not throwing error when field is missing from dict.
Then there's a check if request_id was supplied:
https://github.com/OpenMined/PyGrid/pull/730/files#diff-a7f092ad3c6a3a9977cbc9d3a7c960eaR77-R78

**Review by IonesioJunior — APPROVED** [review]
LGTM!

**Review by cereallarceny — APPROVED** [review]

**Pull request merged**